tgui/yarn.lock

```diff
@@ -4320,6 +4320,24 @@ __metadata:
   languageName: node
   linkType: hard
 
+"fulpui-patches@workspace:packages/fulpui-patches":
```

Comment:
this being in the tgui folder doesnt bother me, but it still feels like there's a ton of TG edits here. Is this part able to be a separate file or is that not how it works? I'm still very new to TGUI

Reply:
This needs to be kept. The tgui package is dependent on it. I tried to do very some hacky stuff within webpack and it was not worth the mess it made.